dnf/rpm/miscutils.py

```diff
@@ -70,7 +70,7 @@ def _verifyPackageUsingRpmkeys(package, installroot):
     # "--define=_pkgverify_flags 0x0" ensures that all signatures and digests
     # are checked.
     args = ('rpmkeys', '--checksig', '--root', installroot, '--verbose',
-            '--define=_pkgverify_level all', '--define=_pkgverify_flags 0x0',
+            '--define=_pkgverify_level signature', '--define=_pkgverify_flags 0x0',
```

Comment:
Please could you also update the comment above the change?

Reply:
Updated.
I have also enhanced the commit message.